src/map.cpp

```diff
@@ -7505,16 +7514,45 @@ void map::draw_rough_circle(std::string type, int x, int y, int rad) {
 void map::draw_rough_circle_furn(furn_id type, int x, int y, int rad) {
     for (int i = x - rad; i <= x + rad; i++) {
         for (int j = y - rad; j <= y + rad; j++) {
-            if (rl_dist(x, y, i, j) + rng(0, 3) <= rad) {
+            if (trig_dist(x, y, i, j) + rng(0, 3) <= rad) {
```

Comment:
rl_dist honors the trigdist / realistic distance setting. with it off (by default), this function previously would draw a square. I assume the mapgens that might use it actually want a visual circle, regardless of movement mechanics.